styles.js

```diff
@@ -9,6 +9,21 @@ export default createGlobalStyle`
 
   body, html {
     margin: 0;
+    padding: 0;
+    height: 100vh;
```

Comment:
Again, see previous comment, but we might want to consider making this a `min-height`.